[OSDOCS#16961]: AUTH-3: Roles, Security Contexts, and Access Policies-CQA-1 (#115746)
openshift/openshift-docs: modules/security-context-constraints-psa-rectifying.adoc

[id="security-context-constraints-psa-rectifying_{context}"]
= About pod security admission alerts

[role="_abstract"]

> If your pods violate the configured pod security standards, you receive a PodSecurityViolation alert that persists for one day so that you can investigate and resolve compliance issues.

Reviewer: ```suggestion
If your pods violate the configured pod security standards, you receive a `PodSecurityViolation` alert that persists for one day so that you can investigate and resolve compliance issues.
```

You can view the Kubernetes API server audit logs to investigate alerts that were triggered. As an example, a workload is likely to fail admission if global enforcement is set to the `restricted` pod security level.

> For assistance in identifying pod security admission violation audit events, see "Audit annotations" in the Kubernetes documentation.

Reviewer: ```suggestion
To identify pod security admission violation audit events, see "Audit annotations" in the Kubernetes documentation.
```
